doc/topics/development/labels.rst

```diff
@@ -0,0 +1,279 @@
+worked on or what might be blocked. All incoming issues must have a milestone associated with them.
+
+Approved
+    Used to indicate that this issue has all of the needed information and is ready to be worked on.
+
+Blocked
+    Used to indicate that the issue is not ready to be worked on yet. This typically applies to issues that have been
+    labeled with “Info Needed”, “Question”, “Expected Behavior”, “Won’t Fix for Now”, etc.
+
+Dot or Bug-fix Release
+    Used to help filter/identify what issues must be fixed before the release such as 2014.7.4 or 2015.2.3. This
+    milestone is often used in conjunction with the ``Blocker`` label, but not always.
+
+Feature Release
+    Similar to the Dot or Bug-fix Release milestone, but for upcoming feature releases such as Boron, Carbon, etc.
+    This milestone is often used in conjunction with the ``Blocker`` label, but not always.
+
+Labels
+======
+
+Labels are used to facilitate the resolution of new pull requests and open issues. Most labels are confined to being
+applied to either issues or pull requests, though some labels may be applied to both.
+
+Issue Labels
+------------
+
+All incoming issues should be triaged with at least one label and a milestone. When a new issue comes in, it should be
+determined if the issue is a bug or a feature request, and either of those labels should be applied accordingly. Bugs
+and Feature Requests have differing labeling schemes, detailed below, where other labels are applied to them to further
+help contributors find issues to fix or implement.
+
+There are some labels, such as ``Question`` or some of the "Status" labels that may be applied as "stand alone" labels
+in which more information may be needed or a decision must be reached on how to proceed. (See the "Bug Status Labels"
+section below.)
+
+Features
+~~~~~~~~
+
+The ``Feature`` label should be applied when a user is requesting entirely new functionality. This can include new
+functions, modules, states, modular systems, flags for existing functions, etc. Features *do not* receive severity
+or priority labels, as those labels are only used for bugs. However, they may receive "Functional Area" labels or "ZD".
+
+Feature request issues will be prioritized on an "as-needed" basis using milestones during SaltStack's feature release
+and sprint planning processes.
+
+Bugs
+~~~~
+
+All bugs should have the ``Bug`` label as well as a severity, priority, functional area, and a status, as applicable.
+
+Severity
+^^^^^^^^
+
+How severe is the bug? SaltStack uses four labels to determine the severity of a bug: ``Blocker``, ``Critical``,
+``High``, and ``Medium``. This scale is intended to make the bug-triage process as objective as possible.
+
+Blocker
+    Should be used sparingly to indicate must-have fixes for the impending release.
+
+Critical
+    Applied to bugs that have data loss, crashes, hanging, unresponsive system, or have no workaround.
```

Comment:
I wouldn't mention workarounds here. We don't want to make any issue without a workaround Critical. *shrugs*